src/garage/misc/process_samples_utils.py

```diff
@@ -0,0 +1,80 @@
+"""Utiliy functions for batch processing samples."""
+from dowel import tabular
+import numpy as np
+
+from garage import log_performance, TrajectoryBatch
+from garage.misc import tensor_utils
+
+
+def preprocess_samples_for_policy_gradient(itr, paths, max_path_length,
```

Comment:
This is also used by CEM and CMA-ES, neither of which are policy gradient algos (my bad). 

Let's be more descriptive, it is basically "paths_to_tensors"